pwiz_tools/Skyline/Model/SrmDocument.cs

```diff
@@ -2772,6 +2772,11 @@ public override int GetHashCode()
             }
         }
 
+        public override string ToString()
+        {
+            return $@"{MoleculeGroupCount},{MoleculeCount},{MoleculeTransitionGroupCount},{MoleculeTransitionCount}"; // For debugging convenience, not user-facing

```

Comment:
Remove this or make it clearer to a developer who has no idea why these 4 values represent SrmDocument.

Reply:
They're the same values in the same order shown in the UI, or used in CheckDocumentState calls. And of course they can check the code if they're confused. I'll add a comment to this effect but would prefer to keep this terse for use as value display in the debugger.

Reply:
I've added this comment:
            // For debugging convenience, not user-facing
            // These are the same values in the same order used to summarize the document in the UI (lower right corner of Skyline window).
            // That's also the same values and order as used in CheckDocumentState() calls.
            // This is terse by design, for ease of display in debugger.